### PR TITLE
Update Prowlarr v0.4.10.2111

### PR DESCRIPTION
**Pull request by wkobiela**
## Description

Update prowlarr - v0.4.10.2111

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Update-Policy#tests-checks)

### Type of change

- [x] Package update 

Tested on DSM7 on virtualDSM and both fresh install and upgrade works fine

### CODE REVIEW

**Comment by hgy59** [comment]
@wkobiela we don't create new packages for packages that use an internal updater. (`SPK_VERS = $(shell date +%Y%m%d)` shows that the package is not bound to a specific version, as it uses an internal updater).

Exceptions of this rule are:
- when the internal updater does not work anymore due to new dependencies
- when new archs are added that are not included in the released packages
- the DSM service integration needs an update (folders, user, port, etc.)


So my proposal for prowlarr:
- update unsupported archs (related to #5383)
- update to use alternate TMP dir (related to #5081)

**Comment by wkobiela** [comment]
It seems that prowlarr is already using alternate TMP dir: https://github.com/SynoCommunity/spksrc/blame/master/spk/prowlarr/Makefile#L38

**Comment by wkobiela** [comment]
@hgy59 do I also need to add `UNSUPPORTED_ARCHS` line in `spk/prowlarr/Makefile`? I've seen it in Radarr files and also in Jellyfin

**Comment by bakerboy448** [comment]
> we don't create new packages for packages that use an internal updater.

This doesn't make any sense for the \*arrs.

So you're saying SynoComm policy is that the \*arrs packages should never be updated?  That means users will always have problems when they reinstall or uninstall and reinstall without purging existing app data.

the old versions that SynoComm provides are not compatible with the new databases and thus the apps will fail to function and users must manually purge their data which is a complicated task for some users

so are there no plans to land this PR then?

**Comment by bakerboy448** [comment]
May as well update this to 0.4.7 rather than having it be an older version.
https://github.com/Prowlarr/Prowlarr/releases

**Review by Safihre — APPROVED** [review]

**Comment by bakerboy448** [comment]
And may as well Bump to 0.4.8 as .2016 had Cardigann issues. Or just land as is and follow up with the updater rework and bump there

**Comment by hgy59** [comment]
@bakerboy448 the automatic update on service start does not work in Prowlarr (it does work in Lidarr).
I have created an issue there (https://github.com/Prowlarr/Prowlarr/issues/1199)

**Comment by hgy59** [comment]
@wkobiela I did some rework. Except the automatic update after package installation or package update, everything seems to be ok.
Can you have a look at it?

File: spk/prowlarr/src/wizard/upgrade_uifile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [{
     "step_title": "Updating Prowlarr",
     "items": [{
-        "desc": "Keep Prowlarr up-to-date by using Prowlarr's built-in updater.<br>Navigate to System>Updates in the Prowlarr UI."
+        "desc": "Keep Prowlarr up-to-date by using Prowlarr's built-in updater.<br>Navigate to System>Updates in the Prowlarr UI.<br/><br/><b style=\"color: red\">Due to an error in Prowlarr, the automatic update in service start does not work, and you have to start the update manually.</b>"
```
Comment (bakerboy448) [inline spk/prowlarr/src/wizard/upgrade_uifile:4]: Id suggest referring to the prowlarr GHI # for easy tracking and future reference

File: spk/prowlarr/src/wizard/upgrade_uifile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [{
     "step_title": "Updating Prowlarr",
     "items": [{
-        "desc": "Keep Prowlarr up-to-date by using Prowlarr's built-in updater.<br>Navigate to System>Updates in the Prowlarr UI.<br/><br/><b style=\"color: red\">Due to an error in Prowlarr, the automatic update in service start does not work, and you have to start the update manually.</b>"
+        "desc": "Keep Prowlarr up-to-date by using Prowlarr's built-in updater.<br>Navigate to System>Updates in the Prowlarr UI.<br/><br/><b style=\"color: red\">Due to an error in Prowlarr, the automatic update in service start does not work, and you have to start the update manually.</b><a href="https://github.com/Prowlarr/Prowlarr/issues/1199">Issue #1199</a>"
```
Comment (bakerboy448) [inline spk/prowlarr/src/wizard/upgrade_uifile:4]: lint failed

`spk/prowlarr/src/wizard/upgrade_uifile: line 4, col 300, found: 'INVALID' - expected: 'EOF', '}', ':', ',', ']'.`

Comment (wkobiela) [reply spk/prowlarr/src/wizard/upgrade_uifile]: Yeah I know, not sure how to include link in upgrade_uifile, probably just lack of backslash

**Comment by bakerboy448** [comment]
the upgrade did not upgrade based on the version in system => status

**Comment by hgy59** [comment]
@wkobiela as we now have the latest version in the package, you cannot see any update in Powlarr.
You only see the file `/var/packages/prowlarr/var/.config/Prowlarr/update_required` that is not deleted as service start.

The automatic update at startup will only work, when there is a version available, that has fixed this and you manually updated to such a version (or we include the fixed version in the package).

But first we need the confirmation that this will be fixed.

**Comment by hgy59** [comment]
As https://github.com/Prowlarr/Prowlarr/issues/1199 is fixed now, we only need to wait for a new release of Prowlarr.

**Comment by wkobiela** [comment]
I'm not sure that automatic update at startup is working, upgraded package manually and Prowlarr is still on previous version
image

**Comment by wkobiela** [comment]
Configured automatic update and updated package manually, still on last version 
image

**Comment by hgy59** [comment]
@wkobiela as I explained in #5511 the automatic update will only work in the future, i.e. when you update the Version 0.4.10.2111 to a future Version.

But you can verfy that the update mechanism is working. When you install v0.4.10.2111 again (over 0.4.10.2111) and the update_required file in the config folder is gone after this "update" than it will work with future versions too.

**Review by hgy59 — APPROVED** [review]

**Pull request merged**